src/config/i18n.js

```diff
@@ -0,0 +1,12 @@
+const MAIN_LANGUAGES = Object.freeze({
```

Comment:
By fetching those values directly in $i18n, you don't need to manually set them here